### ISSUE TITLE
no-class does not catch classes exported with CommonJS

### ISSUE THREAD

**Issue by nickmccurdy** [jfmengels/eslint-plugin-fp#23]
The following example should cause a `no-class` error when enabled, but it doesn't. 
```js
module.exports = class MyClass {}
```

**Comment by macklinu** [comment]
I'll open a PR to address this. 👍
